python/cucim/src/cucim/skimage/measure/_colocalization.py

```diff
@@ -182,6 +185,11 @@ def _get_manders_overlap_coeff(image0, image1):
     return cp.sum(cp.multiply(image0, image1)) / denom
 
 
+def _get_manders_overlap_coeff_no_fusion(image0, image1):
+    denom = (cp.sum(cp.square(image0)) * (cp.sum(cp.square(image1)))) ** 0.5
+    return cp.sum(cp.multiply(image0, image1)) / denom
```

Comment:
Wonder if we can just write this in terms of linalg operations and benefit from BLAS optimizations under-the-hood. Tried this in PR ( https://github.com/rapidsai/cucim/pull/578 ). If we do that, do we still need `fuse`?

Reply:
Yes, that sounds great!